src/main/java/com/mathworks/ci/MatlabRunTestsStepExecution.java

```diff
@@ -68,12 +68,16 @@ private int execMatlabCommand(FilePath workspace, Launcher launcher,
                     "addpath('" + genScriptLocation.getRemote().replaceAll("'", "''") + "'); ";
             final String matlabScriptName = getValidMatlabFileName(genScriptLocation.getBaseName());
 
-            ProcStarter matlabLauncher = getProcessToRunMatlabCommand(workspace, launcher, listener,
-                    envVars, cmdPrefix + matlabScriptName, uniqueTmpFldrName);
+            ProcStarter matlabLauncher =
+                    getProcessToRunMatlabCommand(workspace, launcher, listener, envVars,
+                            cmdPrefix + matlabScriptName + ",delete .matlab/"
+                                    + genScriptLocation.getBaseName() + "/" + matlabScriptName
+                                    + ".m,run(runner_script),rmdir(tmpdir,'s')",
```

Comment:
Need to be careful here. Looks like if `genScriptLocation.getBaseName()` or `matlabScriptName` has a space, this will fail. You may know they won't at the moment, but it's safer to just use the function syntax form of `delete`.

Reply:
These uses 8 bit UUID so i suspect it ever gets space in them.

Reply:
oh ok got the point , are you hinting at use delete() instead delete <file> ?

Reply:
Done.